src/components/Balance.tsx

```diff
@@ -1,40 +1,34 @@
 import React, { useEffect, useRef } from 'react'
 import CountUp from 'react-countup'
-import { Text } from '@pancakeswap/uikit'
-
-interface TextProps {
-  isDisabled?: boolean
-  fontSize?: string
-  color?: string
-  bold?: boolean
-}
+import { Text, TextProps } from '@pancakeswap/uikit'
 
 interface BalanceProps extends TextProps {
-  value?: number
+  value: number
   decimals?: number
   unit?: string
+  isDisabled?: boolean
 }
 
-const Balance: React.FC<BalanceProps> = ({ value, fontSize, color, decimals, isDisabled, unit, bold }) => {
+const Balance: React.FC<BalanceProps> = ({
```

Comment:
Timid refactor of `Balance` component to inherit more from our uikit `TextProps` & `Text` defaults